nativeauthenticator/handlers.py

```diff
@@ -319,12 +381,16 @@ async def post(self):
 
 
 class ChangePasswordAdminHandler(LocalBase):
-    """Render the reset password page."""
+    """Render the /hub/change-password/[someusername] page
+    where admins can set new passwords for users."""
```

Comment:
Similar to ChangePasswordHandler about get/post. This sounds like Get only in a way.

Reply:
See above. I don't really think so, but my perspective isn't the only one. I've made it more explicit.